src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,262 @@
+  </video>  <figcaption class="w-figcaption">
+    Retargeting example, where a mouse click adjusts the animation to a new location. <a href="https://codepen.io/una/details/BaoveQJ">See Demo on Codepen </a>
+  </figcaption>
+</figure>
+
+
+In this example, there is no specified start position. This is an example of using **partial keyframes**. The mouse handler does a few things here: it sets a new end location and triggers a new animation. The new start position is inferred from the current underlying position. 
+
+New transitions can be triggered while existing ones are still running. This means that the current transition is interrupted, and a new one is created.
+
+## Performance improvements with replaceable animations
+
+When creating animations based on events, such as on `"mousemove"`, a new animation is created each time, which can quickly consume memory and degrade performance.  To address this problem, replaceable animations were introduced, enabling automated cleanup, where finished animations are flagged as replaceable and automatically removed if replaced by another finished animation. Consider the following example:
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-animations/comet-trail.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+  A comet trail animates when the mouse moves. <a href="https://codepen.io/kevers-google/details/gOaqXoQ">View Demo on Codepen</a>
+  </figcaption>
+</figure>
+
+ ```js
+elem.addEventListener('mousemove', evt => {
+  rectangle.animate(
+    { transform: translate(${evt.clientX}px, ${evt.clientY}px) },
+    { duration: 500, fill: 'forwards' }
+  );
+});
+```
+
+Each time the mouse moves, the browser re-calculates the position for each ball in the comet trail and creates an animation to this new point. The browser now knows to remove old animations (enabling replacement) when:
+
+1. The animation is finished
+2. There is one or more animations higher in composite ordering that are also finished
+3. The new animations are animating the same properties.
+
+You can see exactly how many animations are being replaced by tallying up a counter with each removed animation, using `"anim.onremove"` to trigger the counter.
+
+There are a few additional methods to take your animation control even further:
+
+- `animation.onremove()` calls some custom JavaScript code when an animation is replaced.
+- `animation.replaceState()` provides a means of tracking whether an animation is active, persisted, or removed.
+- `animation.commitStyles()` updates the style of an element based on the underlying style along with all animations on the element in the composite order.
+- `animation.persist()` marks an animation as non-replaceable.
+
+{% Aside %}
+  The last two methods are commonly used with compositing modes, such as "add". Check out the demo below to see them in action.
```

Comment:
```suggestion
  `animation.commitStyles()` and `animation.persist()` are commonly used with compositing modes, such as "add". Check out the composite modes demo below to see them in action.
```
* Mentioning the methods explicitly will futureproof the article in the event that we update the list but forget to update this aside (it happens all the time)
* Making it clearer what "demo below" we're referring to